rclcpp/include/rclcpp/node.hpp

```diff
@@ -1430,8 +1430,11 @@ class Node : public std::enable_shared_from_this<Node>
   rclcpp::Clock::ConstSharedPtr
   get_clock() const;
 
-  /// Returns current time from the time source specified by clock_type.
+  /// Returns current time from the node time source.
   /**
+   * If `use_sim_time` parameter is set to `true` the simulation time is returned,
+   * otherwise system time is returned.
+   *
```

Comment:
i do not think this is the correct description, if the Node is constructed with `options.clock_type(RCL_SYSTEM_TIME)` or `options.clock_type(RCL_STEADY_TIME)`. i would remove this description.

```suggestion
```